src/commcare_cloud/ansible/group_vars/citusdb_worker.yml

```diff
@@ -1,3 +1,4 @@
 datadog_integration_pgbouncer: false  
 datadog_integration_postgres: true
-is_pg_standby: false
\ No newline at end of file
+is_pg_standby: false
+allow_direct_postgresql_access: true
```

Comment:
this just give `du.user` access right? I don't think we need to do that

Reply:
I also think we can remove this flag since it was just used for running a test prior to moving to RDS

Reply:
I found this flag was being used in pg_hba.conf to enable configs. I used the same flag to enable pg_hba.conf in workers. (Otherwise all connection goes through pgbouncer which is not present on workers. )
https://github.com/dimagi/commcare-cloud/blob/3c540e43839b977ed0fb6d596651a4a9df44a479/src/commcare_cloud/ansible/roles/postgresql/templates/pg_hba.conf.j2#L40

Reply:
I still don't think we should use this flag. There shouldn't be any connections to the workers that aren't coming from the coordinator anyway.